winnaker/helpers.py

```diff
@@ -57,48 +57,53 @@ def a_nice_refresh(driver):
     logging.info("Refreshing the browser ...")
     time.sleep(1)
 
-
-@retry(
-    wait_exponential_multiplier=1000,
-    wait_exponential_max=5000,
-    stop_max_attempt_number=10)
-def wait_for_xpath_presence(driver, xpath, be_clickable=False):
-    logging.debug("Waiting for XPATH: {}".format(xpath))
-    wait = WebDriverWait(driver, 5)
-    try:
-        if be_clickable:
-            e = wait.until(EC.element_to_be_clickable((By.XPATH, xpath)))
-        else:
-            e = wait.until(
-                EC.presence_of_element_located((By.XPATH, xpath)))
-        move_to_element(driver, e)
-        return e
-    except TimeoutException:
-        logging.error("Error: Could not find {}".format(xpath))
-        driver.save_screenshot(
-            join(
-                cfg_output_files_path,
-                "debug_" +
-                now() +
-                ".png"))
-        a_nice_refresh(driver)
-        raise TimeoutException
-    except StaleElementReferenceException:
+def wait_for_xpath_presence(driver,
```

Comment:
Yes this works and we're currently using our fork of Winnaker internally with these code changes.  This needs to be done in this way because of the `retry` decorator.  There is no other way to override the decorator parameters on a per-function call.